config.xml

```diff
@@ -42,11 +42,6 @@
         <allow-intent href="itms:*" />
         <allow-intent href="itms-apps:*" />
         <preference name="AllowBackForwardNavigationGestures" value="true" />
-        <preference name="WKWebViewOnly" value="true" />
-        <feature name="CDVWKWebViewEngine">
-            <param name="ios-package" value="CDVWKWebViewEngine" />
-        </feature>
-        <preference name="CordovaWebViewEngine" value="CDVWKWebViewEngine" />
```

Comment:
Can you provide more info and references on why this is being removed?

Reply:
https://github.com/apache/cordova-plugin-wkwebview-engine/issues/183#issuecomment-754626335